### PR TITLE
Feature/cypress

### PR DESCRIPTION
**Pull request by illusivesunrae**
Installed Cypress and created tests for the Rivet switch add-on.

### CODE REVIEW

**Review by levimcg — CHANGES_REQUESTED** [review]
Nice, this looks great. One small change. We're moving toward all indentation being two spaces. Looks like everything in `switch_spec.js` is four.

You might also look at what it would take to add a couple of assertions that can test the switch custom events that get emitted (`switchOn`, and `switchOff`). Not a directive for this PR, but it could be a nice way to get a bit more familiar with the Cypress API.

File: cypress/integration/switch_spec.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+});
+
+describe("Listen for custom events", function() {
+  it("Listen for the switchOn name", function() {
+    cy.visit(DEV_SERVER);
+    cy.window().then(win => {
+      cy.spy(console, "log");
+      win.addEventListener(win.Switch.on.name, function() {
+        console.log(win.Switch.on.name);
+      });
```
Comment (levimcg) [inline cypress/integration/switch_spec.js:70]: In these blocks and below, let's add some empty lines between statements just to make it more readable. For example, between the `win.addEventListener` and `const toggleSwitch = ...` statements.

Generally speaking, we should add an empty line between each, non-chainable (like Promise `.then()`, or method calls on an Object). I realize there are places above where I broke this rule 😬

**Review by levimcg — CHANGES_REQUESTED** [review]
Good job on the custom event tests. 👍 

Some minor formatting stuff here. Also, while your at it would mind updating the `.gitignore` to match the collapsible? Thanks!

**Review by levimcg — APPROVED** [review]
💯

**Comment by illusivesunrae** [comment]
Addressed issue #9.

**Pull request closed without merging**